brian2/core/network.py

```diff
@@ -865,10 +865,7 @@ def before_run(self, run_namespace):
 
         for obj in all_objects:
             if obj.active:
-                try:
-                    obj.before_run(run_namespace)
-                except Exception as ex:
-                    raise brian_object_exception("An error occurred when preparing an object.", obj, ex)
```

Comment:
What's the thinking here?

Reply:
Oops, I did not mean to commit this change... I used it during debugging because this way the debugger stops at the actual exception and I have access to the full stack frame.